internal/workflow/preprocessing.go

```diff
@@ -662,3 +849,16 @@ func writePREMISFile(ctx temporalsdk_workflow.Context, sip sip.SIP, veraPDFVersi
 
 	return nil
 }
+
+func markdownUList(items []string) string {
```

Comment:
I don't get the name, what's the meaning of the `U`? I also don't see a lot of Markdown, I'd just call it `formatList` or something like that.

Reply:
Markdown Unordered List, like the <ul> HTML element.

Reply:
I changed the function name to `ul()` and added a comment describing its functionality.